### PR TITLE
Jetpack CP: benefits detail UI

### PR DESCRIPTION
**Pull request by jaclync**
Part of #5362 
- [x] ⚠️ Make sure https://github.com/woocommerce/woocommerce-ios/pull/5378 is merged before merging ⚠️ 

## Why

We want to inform users with Jetpack Connection Package (JCP) sites (without Jetpack-the-plugin) about the benefits of installing Jetpack-the-plugin. In the My Store tab, we show a bottom banner for JCP sites with two actions:

- Tapping on the banner shows a screen with Jetpack benefits with a CTA to install Jetpack
- Tapping on the dismiss button dismisses the banner for the site

This PR just contains the UI for the detail screen for easier code review. The display under a feature flag will be in a separate PR.

## Changes

- Added 3 new images: `alarmBellRingImage`, `analyticsImage`, `multipleUsersImage` with PDF assets
- Created `JetpackBenefitsView` with a hosting controller, along with a reusable `JetpackBenefitItem` for each benefit item. The top container is scrollable when needed, for narrower screen height and larger font sizes

## Testing

This UI isn't used yet, feel free to play with SwiftUI previews with more configurations!

## Example screenshots

portrait | landscape
-- | --
Simulator Screen Shot - iPhone 11 - 2021-11-09 at 15 07 07 | Simulator Screen Shot - iPhone 11 - 2021-11-09 at 15 07 15
Simulator Screen Shot - iPhone 11 - 2021-11-09 at 15 07 28 | Simulator Screen Shot - iPhone 11 - 2021-11-09 at 15 07 30

Update release notes:
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by jaclync** [comment]
Added Paolo for code review as well, just one reviewer is needed 😄

File: WooCommerce/Classes/ViewRelated/Dashboard/JetpackConnectionPackageSites/JetpackBenefitItem.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                .frame(width: Layout.circleDimension, height: Layout.circleDimension, alignment: .center)
+                .foregroundColor(Color(.gray(.shade0)))
+                .overlay(
+                    Image(uiImage: icon)
+                        .resizable()
+                        .frame(width: Layout.iconDimension, height: Layout.iconDimension, alignment: .center)
+                )
+            VStack(alignment: .leading) {
+                Text(title).headlineStyle()
+                Spacer().frame(height: Layout.verticalTextSpacing)
```
Comment (ealeksandrov) [inline WooCommerce/Classes/ViewRelated/Dashboard/JetpackConnectionPackageSites/JetpackBenefitItem.swift]: nit: passing spacing as `VStack` parameter will result in same layout without additional `Spacer`.

File: WooCommerce/Classes/ViewRelated/Dashboard/JetpackConnectionPackageSites/JetpackBenefitItem.swift [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    var body: some View {
+        HStack(spacing: Layout.horizontalSpacing) {
+            Circle()
+                .frame(width: Layout.circleDimension, height: Layout.circleDimension, alignment: .center)
+                .foregroundColor(Color(.gray(.shade0)))
+                .overlay(
+                    Image(uiImage: icon)
+                        .resizable()
+                        .frame(width: Layout.iconDimension, height: Layout.iconDimension, alignment: .center)
```
Comment (ealeksandrov) [inline WooCommerce/Classes/ViewRelated/Dashboard/JetpackConnectionPackageSites/JetpackBenefitItem.swift:20]: nit: we can add `@ScaledMetric` property with `* scale` on dimensions to allow Circle and Image grow with dynamic type.

**Review by ealeksandrov — APPROVED** [review]
LGTM! Added couple of minor suggestions :shipit:

File: WooCommerce/Classes/ViewRelated/Dashboard/JetpackConnectionPackageSites/JetpackBenefitItem.swift (hunk 1 above)
Comment (jaclync) [reply WooCommerce/Classes/ViewRelated/Dashboard/JetpackConnectionPackageSites/JetpackBenefitItem.swift]: Ooh I missed that! Updated in https://github.com/woocommerce/woocommerce-ios/pull/5379/commits/671bb4adfa3a8690e2b1c291417b3e3cb0ae3999

File: WooCommerce/Classes/ViewRelated/Dashboard/JetpackConnectionPackageSites/JetpackBenefitItem.swift (hunk 2 above)
Comment (jaclync) [reply WooCommerce/Classes/ViewRelated/Dashboard/JetpackConnectionPackageSites/JetpackBenefitItem.swift:20]: Good point! updated in https://github.com/woocommerce/woocommerce-ios/pull/5379/commits/671bb4adfa3a8690e2b1c291417b3e3cb0ae3999

**Pull request merged**